src/Reduce.cpp

```diff
@@ -17,11 +24,9 @@ void Reduce::reduce(const std::string &key, const std::vector<int> &values) {
 }
 
 void Reduce::export_result(const std::string &key, int value) {
-    // Write result to output directory
-    // Should use File Management Class
-    std::cout << "Writing (" + key + ", " << value << ") to " + this->_output_dir << std::endl;
-
-    // Write SUCCESS file to output directory
-    // Should use File Management Class
-    std::cout << "Writing SUCCESS file to " + this->_output_dir << std::endl;
+    // Write result to output file using fileManager
+    fileManager->appendToFile(
+        this->fileManager->getOutputDirectory(),
+        this->outputFilename,
+        "(" + key + ", " + std::to_string(value) + ")\n");
```

Comment:
I can add a buffer in future PR if we want better performance.

Reply:
I'll resolve this and merge for now, but this is something we can definitely talk about in tonight's meeting.